src/charts/CytoViz/CytoViz.js

```diff
@@ -97,11 +97,19 @@ export const CytoViz = (props) => {
     cytoscapeRef.removeAllListeners();
     // Prevent multiple selection & init elements selection behavior
     cytoscapeRef.on('select', 'node, edge', function (e) {
-      cytoscapeRef.elements().not(e.target).unselect();
+      cytoscapeRef.edges().forEach((edge) => {
+        edge.data({ asInEdgeHighlighted: false, asOutEdgeHighlighted: false });
+      });
       const selectedElement = e.target;
+      selectedElement.select();
+      selectedElement.outgoers('edge').forEach((edge) => edge.data('asOutEdgeHighlighted', true));
+      selectedElement.incomers('edge').forEach((edge) => edge.data('asInEdgeHighlighted', true));
       setCurrentElementDetails(getElementDetailsCallback(selectedElement));
     });
     cytoscapeRef.on('unselect', 'node, edge', function (e) {
+      cytoscapeRef.edges().forEach((edge) => {
+        edge.data({ asInEdgeHighlighted: false, asOutEdgeHighlighted: false });
+      });
```

Comment:
`forEach` loops can be replaced by the `.data(...)` function to make it shorter :
```
cytoscapeRef.edges().data({ asInEdgeHighlighted: false, asOutEdgeHighlighted: false });
[...]
selectedElement.outgoers('edge').data('asOutEdgeHighlighted', true);
selectedElement.incomers('edge').data('asInEdgeHighlighted', true);
[...]
cytoscapeRef.edges().data({ asInEdgeHighlighted: false, asOutEdgeHighlighted: false });
```